File: subplot1.m
t=[0:0.1:3]
x=sin(0.2*pi*t)
y=cos(0.5*pi*t)
z=x+y
subplot(3,1,1)
  plot(x)
subplot(3,1,2)
  plot(y)
subplot(3,1,3)
  plot(z)

File: subplot1.py
import numpy as np
import matplotlib.pyplot as plt
t=np.arange(0,5,0.1)
x=np.sin(0.2*np.pi*t)
y=np.cos(0.5*np.pi*t)
z=x+y
plt.subplot(3,1,1)
plt.plot(x)
plt.subplot(3,1,2)
plt.plot(y)
plt.subplot(3,1,3)
plt.plot(z)
plt.show()